apps/src/tutorialExplorer/tutorialDetail.jsx

```diff
@@ -355,6 +355,10 @@ const styles = {
   tutorialDetailDisabledIcon: {
     color: '#d9534f'
   },
+  tutorialDetailLanguages: {
+    overflowY: 'auto',
+    maxHeight: '110px'
+  },
```

Comment:
Is this used anywhere?

Reply:
We should definitely remove this. It is a remnant style of our original solution. Here is a link to [the PR](https://github.com/code-dot-org/code-dot-org/pull/43466) for clean up regarding this work.